### ISSUE TITLE
Change f_key name to 'organization_id', from 'organization_uuid'

### ISSUE THREAD

**Issue by jinjagit** [liquidvotingio/api#74]
Will need coordinating with [#28](https://github.com/liquidvotingio/auth/issues/28)  (auth: "change p_key name to 'id' from 'uuid'")

**Comment by oliverbarnes** [comment]
Just a note, this change isn't dependent on #28 per se. They _are_ dependent on the mapping done on https://github.com/liquidvotingio/api/blob/master/lib/liquid_voting_web/plugs/context.ex#L17-L19

```
  def build_context(conn) do
    with [organization_uuid] <- get_req_header(conn, "org-uuid") do
      %{organization_uuid: organization_uuid}
```

And then updating how that context gets passed around to resolvers like https://github.com/liquidvotingio/api/blob/master/lib/liquid_voting_web/resolvers/voting.ex#L11:

```
def create_participant(_, args, %{context: %{organization_uuid: organization_uuid}}) do
```

Updating the headers will have to be coordinated across different repos though, not only `api` and `auth`, but `ruby-client` and `deployment` as well - the kubernetes ingress configures the header the auth forwards to api:  https://github.com/liquidvotingio/deployment/blob/master/k8s/api-ingress.yaml#L8

```
nginx.ingress.kubernetes.io/auth-response-headers: Org-UUID, X-Request-ID
```

**Comment by jinjagit** [comment]
Closed, since this was covered in [#76](https://github.com/liquidvotingio/api/pull/76).
